node_modules/oae-core/tenantlandingpage/js/tenantlandingpage.js

```diff
@@ -40,6 +40,18 @@ define(['jquery', 'oae.core'], function($, oae) {
             $.ajax({
                 'url': '/api/tenant/landingPage',
                 'success': function(blocks) {
+                    // Replace all instances of ${tenantDisplayName} in block
```

Comment:
Would you be calling Trimpath directly or do a utility function wrapper around it?